exercises/tuples-lists-dictionaries/challenge_072.py

```diff
@@ -6,9 +6,27 @@
 """
 
 subjects_list = ["Maths", "Literature", "Science", "History", "Spanish", "Computing"]
-print(subjects_list)
-dislike = input("Which of these subjects do you dislike? ")
-erase_from_list = subjects_list.index(dislike)
-del subjects_list[erase_from_list]
-print(subjects_list)
 
+
+def join_subjects(subjects_list):
+    return ", ".join(subjects_list)
+
+
+print(f"Subjects list: {join_subjects(subjects_list)}")
+
+
+def ask_user_input():
+    while True:
+        user_input = input("Which one of these subjects do you dislike? ")
+        if user_input in subjects_list:
+            erase_from_list = subjects_list.index(user_input)
+            del subjects_list[erase_from_list]
```

Comment:
There is still a problem here if the user enters invalid input. Try entering an invalid input and seeing what happens.

Reply:
I have updated the challenge to include more subjects and also I am checking for lowercase inputs. Everything seems to be working now.